config/locales/devise.hi.yml

```diff
@@ -0,0 +1,60 @@
+      send_paranoid_instructions: 'यदि आपका ईमेल हमारे डेटाबेस में मौजूद है, तो आप कुछ ही मिनटों में अपने ईमेल पते की पुष्टि करने के निर्देश के साथ एक ईमेल प्राप्त करेंगे।'
+    failure:
+      already_authenticated: 'आप पहले ही साइन इन कर चुके हैं।'
+      inactive: 'आपका खाता अभी तक सक्रिय नहीं किया गया है।'
+      invalid: 'अमान्य %{authentication_keys} या पासवर्ड।'
+      locked: 'आपका खाता लॉक है।'
+      last_attempt: 'आपके खाते के लॉक होने से पहले आपके पास एक और प्रयास है।'
+      not_found_in_database: 'अमान्य %{authentication_keys} या पासवर्ड।'
+      timeout: 'आपका सत्र समाप्त हो गया। जारी रखने के लिए कृपया पुन: प्रवेश करें।'
+      unauthenticated: 'जारी रखने से पहले आपको साइन इन या साइन अप करने की आवश्यकता है।'
+      unconfirmed: 'जारी रखने से पहले आपको अपने ईमेल पते की पुष्टि करनी होगी।'
+    mailer:
+      confirmation_instructions:
+        subject: 'पुष्टि निर्देश'
+      reset_password_instructions:
+        subject: 'पासवर्ड रीसेट करने के निर्देश'
+      unlock_instructions:
+        subject: 'अनलॉक करने के निर्देश'
+      password_change:
+        subject: 'पासवर्ड बदला गया'
+    omniauth_callbacks:
+      failure: 'आपको %{kind} से प्रमाणित नहीं किया जा सकता क्योंकि %{reason}।'
+      success: '%{reason} खाते से सफलतापूर्वक प्रमाणित किया गया।'
+    passwords:
+      no_token: 'आप पासवर्ड रीसेट ईमेल से आए बिना इस पृष्ठ तक नहीं पहुंच सकते। यदि आप एक पासवर्ड रीसेट ईमेल से आते हैं, तो कृपया सुनिश्चित करें कि आपने पूर्ण URL प्रदान किया है।'
+      send_instructions: 'आपको कुछ मिनटों में अपना पासवर्ड रीसेट करने के निर्देशों के साथ एक ईमेल प्राप्त होगा।'
+      send_paranoid_instructions: 'यदि आपका ईमेल पता हमारे डेटाबेस में मौजूद है, तो आपको कुछ ही मिनटों में आपके ईमेल पते पर एक पासवर्ड रिकवरी लिंक प्राप्त होगा।'
+      updated: 'आपका पासवर्ड सफलतापूर्वक बदल दिया गया है। अब आप साइन इन हैं।'
+      updated_not_active: 'आपका पासवर्ड सफलतापूर्वक बदल दिया गया है।'
+    registrations:
+      destroyed: 'अलविदा! आपका खाता सफलतापूर्वक रद्द कर दिया गया है। हम जल्द ही फिर से मिलने की उम्मीद करते है।'
+      signed_up: 'स्वागत है! आपने सफलतापूर्वक साइन अप कर लिया है।'
+      signed_up_but_inactive: 'आपने सफलतापूर्वक साइन अप कर लिया है। हालाँकि, हम आपको साइन नहीं कर सकते क्योंकि आपका खाता अभी तक सक्रिय नहीं है।'
+      signed_up_but_locked: 'आपने सफलतापूर्वक साइन अप कर लिया है। हालाँकि, हम आपको साइन नहीं कर सके क्योंकि आपका खाता बंद है।'
+      signed_up_but_unconfirmed: 'आपके ईमेल पते पर एक पुष्टिकरण लिंक के साथ एक संदेश भेजा गया है। कृपया अपने खाते को सक्रिय करने के लिए लिंक का अनुसरण करें।'
+      update_needs_confirmation: 'आपने अपना खाता सफलतापूर्वक अपडेट कर दिया है, लेकिन हमें आपके नए ईमेल पते को सत्यापित करना होगा। कृपया अपने ईमेल की जाँच करें और अपने नए ईमेल पते की पुष्टि करने के लिए पुष्टि लिंक का पालन करें।'
+      updated: 'आपका खाता सफलतापूर्वक अपडेट कर दिया गया है।'
+    sessions:
+      signed_in: 'सफलतापूर्वक हस्ताक्षर किए गए।'
+      signed_out: 'सफलतापूर्वक हस्ताक्षर किए गए।'
```

Comment:
Maybe just "साइन आउट" here

Reply:
Maybe we should do same for signed in too. Suggestions?

Reply:
> Maybe we should do same for signed in too. 
I agree. I believe most will get confused by "हस्ताक्षर".